eo-runtime/src/test/java/EOorg/EOeolang/EOcageTest.java

```diff
@@ -170,9 +171,9 @@ void evaluatesLazily() {
         EOcageTest.writeTo(first, new Data.ToPhi(3L));
         final Phi second = new EOcage(Phi.Φ);
         EOcageTest.writeTo(second, new Data.ToPhi(5L));
-        final Phi cage = new EOcage(Phi.Φ);
+        final Phi sum = new EOcage(Phi.Φ);
```

Comment:
@levBagryansky Why have you decided to rename `cage` -> `sum`? `cage` seems more logical name.

Reply:
@levBagryansky Agree with you here. Thank you for the explanation.